terraformer_ec2.tf

```diff
@@ -0,0 +1,72 @@
+# The Terraformer AMI
+data "aws_ami" "terraformer" {
+  provider = aws.provisionassessment
+
+  filter {
+    name = "name"
+    values = [
+      "terraformer-hvm-*-x86_64-ebs"
+    ]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  filter {
+    name   = "root-device-type"
+    values = ["ebs"]
+  }
+
+  owners      = [local.images_account_id]
+  most_recent = true
+}
+
+# The Terraformer EC2 instances
+resource "aws_instance" "terraformer" {
+  count    = lookup(var.operations_instance_counts, "terraformer", 0)
+  provider = aws.provisionassessment
+
+  ami                  = data.aws_ami.terraformer.id
+  availability_zone    = "${var.aws_region}${var.aws_availability_zone}"
+  iam_instance_profile = aws_iam_instance_profile.terraformer.name
+  instance_type        = "t3.medium"
+  # For some reason I can't SSM to the instance unless I put it in the
+  # first private subnet.  I believe this has something to do with the
+  # NACLs that are in place for that subnet specifically.  I will
+  # figure this out later.  I'd definitely prefer to put this instance
+  # in a different subnet than the Guacamole instance.
```

Comment:
It sounds like this should be a `TODO` with a new issue to go along with it.

Reply:
See commit 5bc0ddb.